examples/data-sources/twingate_remote_network/data-source.tf

```diff
@@ -0,0 +1,17 @@
+provider "twingate" {}
+
+resource "twingate_remote_network" "net1" {
```

Comment:
Looking at docs for others datasources (like [here](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/pubsub_topic) and [here](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/data_source_dataproc_metastore_service)) they dont include the resource creation in the example.

It makes sense because the example doesn't make sense - if you create it via TF you dont need to reference it by name via datasource.

Any reason this example shouldn't just be:
```
data "twingate_remote_network" "foo" {
   id = "<your network's id>"
 }
```

Reply:
sure, we can do like that, 
my intention was to provide working example, but your proposal also make sense

will update it

Reply:
@ekampf updated example